database/buildSchema/27_review.sql

```diff
@@ -17,7 +17,9 @@ CREATE TABLE public.review (
 	-- status via review_status_history
 	trigger public.trigger,
 	application_id integer GENERATED ALWAYS AS (public.review_application_id(review_assignment_id)) STORED references public.application(id),
-	reviewer_id integer GENERATED ALWAYS AS (public.review_reviewer_id(review_assignment_id)) STORED references public.user(id)
+	reviewer_id integer GENERATED ALWAYS AS (public.review_reviewer_id(review_assignment_id)) STORED references public.user(id),
+	level integer,
+	can_make_overall_decision boolean
```

Comment:
These should be generated columns derived from the `review_assignment` table. You probably want to merge this PR soon, so can you please make a new issue to add these two as generated columns?

Reply:
Yep, not problem. I'll add a new small issue to do that.